src/components/Editor/Menu/Fixed/components/EmojiOption.jsx

```diff
@@ -21,6 +21,7 @@ const EmojiOption = ({
     dropdownProps={{ classNames: "neeto-editor-fixed-menu__emoji-dropdown" }}
     icon={Smiley}
     isOpen={isActive}
+    label="emoji"
```

Comment:
![image](https://github.com/bigbinary/neeto-editor/assets/35297280/16b10843-4940-4096-a5a8-85acf04a667e)

We do not need a label for the Emoji option. The icon is self explanatory.